src/background/utils/overwriteWitnetNodeConfiguration.ts

```diff
@@ -1,30 +1,60 @@
 import path from 'path'
 import fs from 'fs-extra'
 
+const nodeUrlUntilCharacter = (character: string) =>
+  new RegExp('node_url =([^;]*)' + character)
+
+const nodeUrlsRegex = /g\[([^;]*)\]|\"([^;]*)\"/
+
+export function hasOldWitnetIp({
+  oldWitnetNodes,
+  currentNodeConfiguration,
+}: {
+  oldWitnetNodes: Array<string>
+  currentNodeConfiguration: Array<string>
+}) {
+  return currentNodeConfiguration.some(ip => oldWitnetNodes.includes(ip))
+}
```

Comment:
This function doesn't need to know about our business  logic
```suggestion
export function existIntersection(
  arr1: Array<string>,
  arr2: Array<string>,
)) {
  return arr1.some(item => arr2.includes(item))
}
```